src/components/BrowseRequests.jsx

```diff
@@ -1,13 +1,29 @@
 import React from "react";
-import { Grid } from "semantic-ui-react";
+import { Grid, List } from "semantic-ui-react";
 import AllRequests from "./AllRequests";
```

Comment:
I think your changes is supposed to be inside of this AllRequests component which was rendered here previously